msm-sitemap.php

```diff
@@ -397,10 +401,12 @@ public static function get_post_ids_for_date( $sitemap_date, $limit = 500 ) {
 		global $wpdb;
 
 		$start_date = $sitemap_date . ' 00:00:00';
-		$end_date = $sitemap_date . ' 23:59:59';
-		$post_types_in = self::get_supported_post_types_in();
+		$end_dat    = $sitemap_date . ' 23:59:59';
```

Comment:
There is a spelling mistake in the variable name here so the end_date variable below is not utilised appropriately

Reply:
Ah, I didn't see it because it was not found on line 404 😄